test/Microsoft.ML.Tests/PermutationFeatureImportanceTests.cs

```diff
@@ -325,8 +325,8 @@ public void TestPfiClusteringOnDenseFeatures()
 
             var preview = data.Preview();
 
-            var model = ML.Clustering.Trainers.KMeans("Features", clustersCount: 5, 
-                advancedSettings: args =>{ args.NormalizeFeatures = NormalizeOption.No;})
+            var model = ML.Clustering.Trainers.KMeans("Features", clustersCount: 5,
+                advancedSettings: args =>{ args.NormalizeFeatures = NormalizeOption.No; args.NumThreads = 1; })
```

Comment:
This is what I am hoping will solve the issue.

Reply:
I am also not sure what is this `NormalizeOption.No`. I do not think `Fit` will auto-normalize in any event anyway?

Reply:
Yep, I think this argument is only use by the entry point.

---
In reply to: [242977200](https://github.com/dotnet/machinelearning/pull/1926#discussion_r242977200) [](ancestors = 242977200)